.travis.yml

```diff
@@ -34,6 +34,12 @@ matrix:
       env: BEAKER_set="centos-6" BEAKER_PUPPET_COLLECTION="puppet5"
       bundler_args:
       script: bundle exec rake beaker
+    - rvm: 2.4.4
+      sudo: required
+      services: docker
+      env: BEAKER_set="centos-7" BEAKER_PUPPET_COLLECTION="puppet5" BEAKER_yum_full="yes"
```

Comment:
can we get rid of the other centos-7 test then?

Reply:
Actually used wrong set , updated PR to use correct new set that contains multiple nodes.